### ISSUE TITLE
1.0.2 crashes on 1.20.1

### ISSUE THREAD

**Issue by Felix14-v2** [JustAlittleWolf/centered-crosshair#1]
### Description
Well, it just crashes, But this version is marked as compatible on Modrinth. 1.0.1 works fine.
![image](https://github.com/JustAlittleWolf/centered-crosshair/assets/75726196/3ffd4c06-0784-419c-b1bb-b481bb038a7b)

### To reproduce
- Make a clear instance with 1.20.1
- Install Centered Crosshair 1.0.2
- Try launching the game
- See the crash

### Logs
<details>
 <summary>View</summary>

```
[23:50:55] [main/INFO]: Loading Minecraft 1.20.1 with Fabric Loader 0.14.22
[23:50:55] [main/INFO]: Loading 66 mods:
	- architectury 9.1.12
	- centered-crosshair 1.0.2
	- cloth-config 11.1.106
	   \-- cloth-basic-math 0.6.1
	- enhanced_searchability 1.0.0
	- fabric-api 0.88.1+1.20.1
	   |-- fabric-api-base 0.4.30+7abfd51577
	   |-- fabric-api-lookup-api-v1 1.6.35+4d8536c977
	   |-- fabric-biome-api-v1 13.0.10+b3afc78b77
	   |-- fabric-block-api-v1 1.0.10+92a0d36777
	   |-- fabric-block-view-api-v2 1.0.0+92a0d36777
	   |-- fabric-blockrenderlayer-v1 1.1.40+b3afc78b77
	   |-- fabric-client-tags-api-v1 1.1.1+97bb207577
	   |-- fabric-command-api-v1 1.2.33+f71b366f77
	   |-- fabric-command-api-v2 2.2.12+b3afc78b77
	   |-- fabric-commands-v0 0.2.50+df3654b377
	   |-- fabric-containers-v0 0.1.63+df3654b377
	   |-- fabric-content-registries-v0 4.0.9+b3afc78b77
	   |-- fabric-convention-tags-v1 1.5.4+a1a980da77
	   |-- fabric-crash-report-info-v1 0.2.18+aeb40ebe77
	   |-- fabric-data-generation-api-v1 12.2.2+1e61dba177
	   |-- fabric-dimensions-v1 2.1.53+8536527b77
	   |-- fabric-entity-events-v1 1.5.22+b3afc78b77
	   |-- fabric-events-interaction-v0 0.6.1+e91849a877
	   |-- fabric-events-lifecycle-v0 0.2.62+df3654b377
	   |-- fabric-game-rule-api-v1 1.0.38+b04edc7a77
	   |-- fabric-item-api-v1 2.1.27+b3afc78b77
	   |-- fabric-item-group-api-v1 4.0.10+23d9108177
	   |-- fabric-key-binding-api-v1 1.0.36+fb8d95da77
	   |-- fabric-keybindings-v0 0.2.34+df3654b377
	   |-- fabric-lifecycle-events-v1 2.2.21+b3afc78b77
	   |-- fabric-loot-api-v2 1.1.39+b3afc78b77
	   |-- fabric-loot-tables-v1 1.1.43+9e7660c677
	   |-- fabric-message-api-v1 5.1.7+3265161977
	   |-- fabric-mining-level-api-v1 2.1.49+b3afc78b77
	   |-- fabric-model-loading-api-v1 1.0.2+709a987177
	   |-- fabric-models-v0 0.4.1+9386d8a777
	   |-- fabric-networking-api-v1 1.3.10+eeb8eb3677
	   |-- fabric-networking-v0 0.3.50+df3654b377
	   |-- fabric-object-builder-api-v1 11.1.1+6beca84877
	   |-- fabric-particles-v1 1.1.1+201a23a077
	   |-- fabric-recipe-api-v1 1.0.20+b3afc78b77
	   |-- fabric-registry-sync-v0 2.3.2+4df89eb277
	   |-- fabric-renderer-api-v1 3.1.3+92a0d36777
	   |-- fabric-renderer-indigo 1.4.3+6bdb2ed077
	   |-- fabric-renderer-registries-v1 3.2.45+df3654b377
	   |-- fabric-rendering-data-attachment-v1 0.3.36+92a0d36777
	   |-- fabric-rendering-fluids-v1 3.0.27+b3afc78b77
	   |-- fabric-rendering-v0 1.1.48+df3654b377
	   |-- fabric-rendering-v1 3.0.7+b3afc78b77
	   |-- fabric-resource-conditions-api-v1 2.3.5+ea08f9d877
	   |-- fabric-resource-loader-v0 0.11.9+132c48c177
	   |-- fabric-screen-api-v1 2.0.7+b3afc78b77
	   |-- fabric-screen-handler-api-v1 1.3.29+b3afc78b77
	   |-- fabric-sound-api-v1 1.0.12+b3afc78b77
	   |-- fabric-transfer-api-v1 3.3.1+195226a777
	   \-- fabric-transitive-access-wideners-v1 4.3.0+6c31357e77
	- fabricloader 0.14.22
	- java 17
	- minecraft 1.20.1
	- recursiveresources 2.5.2+1.20
	   \-- cicada 0.4.0
	        \-- com_github_llamalad7_mixinextras 0.2.0-beta.6
	- shared-resources 1.6.0
	   |-- shared-resources-api 1.6.0
	   \-- shared-resources-mc20 1.6.0
[23:50:55] [main/INFO]: SpongePowered MIXIN Subsystem Version=0.8.5 Source=file:/C:/Users/%d0%94%d0%be%d0%bc%d0%b0%d1%88%d0%bd%d0%b8%d0%b9%20%d0%ba%d0%be%d0%bc%d0%bf%d1%8c%d1%8e%d1%82%d0%b5%d1%80/AppData/Roaming/com.modrinth.theseus/meta/libraries/net/fabricmc/sponge-mixin/0.12.5+mixin.0.8.5/sponge-mixin-0.12.5+mixin.0.8.5.jar Service=Knot/Fabric Env=CLIENT
[23:50:56] [main/INFO]: Compatibility level set to JAVA_16
[23:50:56] [main/INFO]: Compatibility level set to JAVA_17
[23:50:56] [main/WARN]: Reference map 'enhanced_searchability-fabric-refmap.json' for fabric-enhanced_searchability.mixins.json could not be read. If this is a development environment you can ignore this message
[23:50:56] [main/WARN]: Reference map 'shared-resources-api-refmap.json' for shared-resources-api.mixins.json could not be read. If this is a development environment you can ignore this message
[23:50:57] [main/WARN]: Error loading class: net/minecraft/client/util/ScreenshotUtils (java.lang.ClassNotFoundException: net/minecraft/client/util/ScreenshotUtils)
[23:50:57] [main/WARN]: @Mixin target net/minecraft/client/util/ScreenshotUtils was not found shared-resources.mixins.json:screenshots.ScreenshotUtilsMixin from mod shared-resources
[23:51:05] [Datafixer Bootstrap/INFO]: 188 Datafixer optimizations took 295 milliseconds
[23:51:08] [Render thread/INFO]: [STDERR]: Version: 3.3.1 SNAPSHOT
[23:51:08] [Render thread/INFO]: [STDERR]: 	 OS: Windows 11 v10.0
[23:51:08] [Render thread/INFO]: [STDERR]: 	JRE: Windows amd64 17.0.3
[23:51:08] [Render thread/INFO]: [STDERR]: 	JVM: OpenJDK 64-Bit Server VM v17.0.3+7-LTS by Microsoft
[23:51:08] [Render thread/INFO]: [STDERR]: Loading JNI library: lwjgl
[23:51:08] [Render thread/INFO]: [STDERR]: 	Module: org.lwjgl
[23:51:08] [Render thread/INFO]: [STDERR]: 	Loaded from org.lwjgl.librarypath: C:\Users\Домашний компьютер\AppData\Roaming\com.modrinth.theseus\meta\natives\1.20.1-0.14.22\lwjgl.dll
[23:51:08] [Render thread/INFO]: [STDERR]: Loading library: jemalloc
[23:51:08] [Render thread/INFO]: [STDERR]: 	Module: org.lwjgl.jemalloc
[23:51:08] [Render thread/INFO]: [STDERR]: 	Loaded from org.lwjgl.librarypath: C:\Users\Домашний компьютер\AppData\Roaming\com.modrinth.theseus\meta\natives\1.20.1-0.14.22\jemalloc.dll
[23:51:08] [Render thread/INFO]: [STDERR]: MemoryUtil allocator: JEmallocAllocator
[23:51:08] [Render thread/INFO]: Environment: authHost='https://authserver.mojang.com', accountsHost='https://api.mojang.com', sessionHost='https://sessionserver.mojang.com', servicesHost='https://api.minecraftservices.com', name='PROD'
[23:51:09] [Render thread/INFO]: Setting user: Felix14_v2
[23:51:09] [Render thread/INFO]: Registering S2C receiver with id architectury:spawn_entity_packet
[23:51:10] [Render thread/INFO]: [Indigo] Registering Indigo renderer!
[23:51:10] [Cicada thread 0/INFO]: [cicada] Hello, anyone there?
[23:51:10] [Cicada thread 0/INFO]: [recursiveresources] Hallo, I'm here!
[23:51:10] [Render thread/INFO]: [STDERR]: Loading library: glfw
[23:51:10] [Render thread/INFO]: [STDERR]: 	Module: org.lwjgl.glfw
[23:51:10] [Render thread/INFO]: [STDERR]: 	Loaded from org.lwjgl.librarypath: C:\Users\Домашний компьютер\AppData\Roaming\com.modrinth.theseus\meta\natives\1.20.1-0.14.22\glfw.dll
[23:51:10] [Render thread/INFO]: Backend library: LWJGL version 3.3.1 SNAPSHOT
[23:51:10] [Render thread/INFO]: [STDERR]: Java 9 check intrinsics enabled
[23:51:10] [Render thread/INFO]: [STDERR]: Closure Registry: simple
[23:51:11] [Render thread/INFO]: [STDERR]: Loading JNI library: lwjgl_opengl
[23:51:11] [Render thread/INFO]: [STDERR]: 	Module: org.lwjgl.opengl
[23:51:11] [Render thread/INFO]: [STDERR]: 	Loaded from org.lwjgl.librarypath: C:\Users\Домашний компьютер\AppData\Roaming\com.modrinth.theseus\meta\natives\1.20.1-0.14.22\lwjgl_opengl.dll
[23:51:11] [Render thread/INFO]: [STDERR]: Loading library: opengl32
[23:51:11] [Render thread/INFO]: [STDERR]: 	Module: org.lwjgl.opengl
[23:51:11] [Render thread/INFO]: [STDERR]: 	opengl32.dll not found in org.lwjgl.librarypath=C:\Users\Домашний компьютер\AppData\Roaming\com.modrinth.theseus\meta\natives\1.20.1-0.14.22
[23:51:11] [Render thread/INFO]: [STDERR]: 	Loaded from system paths: C:\Windows\SYSTEM32\opengl32.dll
[23:51:11] [Render thread/INFO]: [STDERR]: Java 10 multiplyHigh enabled
[23:51:11] [Render thread/INFO]: [STDERR]: [GL] GL_ARB_shader_subroutine was reported as available but an entry point is missing.
[23:51:11] [Render thread/INFO]: [STDERR]: Loading JNI library: lwjgl_stb
[23:51:11] [Render thread/INFO]: [STDERR]: 	Module: org.lwjgl.stb
[23:51:11] [Render thread/INFO]: [STDERR]: 	Loaded from org.lwjgl.librarypath: C:\Users\Домашний компьютер\AppData\Roaming\com.modrinth.theseus\meta\natives\1.20.1-0.14.22\lwjgl_stb.dll
[23:51:11] [Render thread/INFO]: [STDERR]: Java 9 text decoding enabled
[23:51:11] [Render thread/WARN]: Missing metadata in pack file/Aquatic Wars III.zip
[23:51:11] [Render thread/WARN]: Missing metadata in pack file/1.17.1/Better netherite v2.zip
[23:51:11] [Render thread/WARN]: Missing metadata in pack file/1.17.1/snod-torches.zip
[23:51:11] [Render thread/WARN]: Missing metadata in pack file/1.18.2/Faithful-Java-32x-1.18.2.zip
[23:51:11] [Render thread/WARN]: Missing metadata in pack file/1.18.2/spherical-ghasts.zip
[23:51:11] [Render thread/ERROR]: Failed to parse allowed_symlinks.txt, disallowing all symbolic links
java.nio.file.AccessDeniedException: C:\Users\Домашний компьютер\AppData\Roaming\.minecraft\home\Общий Minecraft\saves
	at sun.nio.fs.WindowsException.translateToIOException(WindowsException.java:89) ~[?:?]
	at sun.nio.fs.WindowsException.rethrowAsIOException(WindowsException.java:103) ~[?:?]
	at sun.nio.fs.WindowsException.rethrowAsIOException(WindowsException.java:108) ~[?:?]
	at sun.nio.fs.WindowsFileSystemProvider.newByteChannel(WindowsFileSystemProvider.java:236) ~[?:?]
	at java.nio.file.Files.newByteChannel(Files.java:380) ~[?:?]
	at java.nio.file.Files.newByteChannel(Files.java:432) ~[?:?]
	at java.nio.file.spi.FileSystemProvider.newInputStream(FileSystemProvider.java:422) ~[?:?]
	at java.nio.file.Files.newInputStream(Files.java:160) ~[?:?]
	at java.nio.file.Files.newBufferedReader(Files.java:2922) ~[?:?]
	at java.nio.file.Files.newBufferedReader(Files.java:2955) ~[?:?]
	at net.minecraft.class_32.method_52235(class_32.java:128) ~[client-intermediary.jar:?]
	at net.minecraft.class_310.<init>(class_310.java:525) ~[client-intermediary.jar:?]
	at net.minecraft.client.main.Main.main(Main.java:211) ~[1.20.1-0.14.22.jar:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:468) ~[fabric-loader-0.14.22.jar:?]
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74) ~[fabric-loader-0.14.22.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23) ~[fabric-loader-0.14.22.jar:?]
[23:51:12] [Render thread/INFO]: [STDERR]: Loading library: OpenAL
[23:51:12] [Render thread/INFO]: [STDERR]: 	Module: org.lwjgl.openal
[23:51:12] [Render thread/INFO]: [STDERR]: 	Loaded from org.lwjgl.librarypath: C:\Users\Домашний компьютер\AppData\Roaming\com.modrinth.theseus\meta\natives\1.20.1-0.14.22\OpenAL.dll
[23:51:12] [Render thread/ERROR]: Mixin apply for mod centered-crosshair failed centered-crosshair.mixins.json:InGameHudMixin from mod centered-crosshair -> net.minecraft.class_329: org.spongepowered.asm.mixin.transformer.throwables.InvalidMixinException @Shadow field ICONS was not located in the target class net.minecraft.class_329. Using refmap centered-crosshair-refmap.json
org.spongepowered.asm.mixin.transformer.throwables.InvalidMixinException: @Shadow field ICONS was not located in the target class net.minecraft.class_329. Using refmap centered-crosshair-refmap.json
	at org.spongepowered.asm.mixin.transformer.MixinPreProcessorStandard.attachFields(MixinPreProcessorStandard.java:624) ~[sponge-mixin-0.12.5+mixin.0.8.5.jar:0.12.5+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinPreProcessorStandard.attach(MixinPreProcessorStandard.java:302) ~[sponge-mixin-0.12.5+mixin.0.8.5.jar:0.12.5+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinPreProcessorStandard.createContextFor(MixinPreProcessorStandard.java:277) ~[sponge-mixin-0.12.5+mixin.0.8.5.jar:0.12.5+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinInfo.createContextFor(MixinInfo.java:1289) ~[sponge-mixin-0.12.5+mixin.0.8.5.jar:0.12.5+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.apply(MixinApplicatorStandard.java:294) ~[sponge-mixin-0.12.5+mixin.0.8.5.jar:0.12.5+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.TargetClassContext.apply(TargetClassContext.java:421) ~[sponge-mixin-0.12.5+mixin.0.8.5.jar:0.12.5+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.TargetClassContext.applyMixins(TargetClassContext.java:403) ~[sponge-mixin-0.12.5+mixin.0.8.5.jar:0.12.5+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:363) ~[sponge-mixin-0.12.5+mixin.0.8.5.jar:0.12.5+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClass(MixinTransformer.java:234) ~[sponge-mixin-0.12.5+mixin.0.8.5.jar:0.12.5+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClassBytes(MixinTransformer.java:202) ~[sponge-mixin-0.12.5+mixin.0.8.5.jar:0.12.5+mixin.0.8.5]
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.getPostMixinClassByteArray(KnotClassDelegate.java:422) ~[fabric-loader-0.14.22.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.tryLoadClass(KnotClassDelegate.java:323) ~[fabric-loader-0.14.22.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.loadClass(KnotClassDelegate.java:218) ~[fabric-loader-0.14.22.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:112) ~[fabric-loader-0.14.22.jar:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:520) ~[?:?]
	at net.minecraft.class_310.<init>(class_310.java:603) ~[client-intermediary.jar:?]
	at net.minecraft.client.main.Main.main(Main.java:211) ~[1.20.1-0.14.22.jar:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:468) ~[fabric-loader-0.14.22.jar:?]
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74) ~[fabric-loader-0.14.22.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23) ~[fabric-loader-0.14.22.jar:?]
```
</details>

[2023-09-26-1.log](https://github.com/JustAlittleWolf/centered-crosshair/files/12740238/2023-09-26-1.log)

**Comment by JustAlittleWolf** [comment]
Thanks for opening the issue, I will look into it. For now you can use the 1.0.1 version, the only thing that changed from that one and 1.0.2 is 1.20.2 support anyways.

**Comment by JustAlittleWolf** [comment]
Should be fixed now! The new version is only for 1.20.2, so the old version 1.0.1 is the way to go for 1.20.1.
